src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.reader.CsvFileReader;
+import core.basesyntax.report.CsvFileWriter;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvFileReader csvFileReader = new CsvFileReader("testData1.csv");
```

Comment:
Don't ignore exceptions in try-catch blocks. Make sure to handle exceptions properly by rethrowing them with an informative message.